eeps/0003-normalize-path-to-attrs.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: Align file paths to the exposed attrs
+Author: jonringer
+Discussions-To: 
+Status: Draft
+Type: Standards Track
+Repo: All poly repos
+Topic: Packaging
+Created: 2024-10-06
+---
+
+# Summary
+
+Nixpkgs contains a lot of legacy "oddities" which cause some impediance mismatch
+when currating a large package set. One of these "oddities" is that is fairly
```

Comment:
I was just "drive-by-reading" and saw these... they're just two spelling nitpicks.

```suggestion
when curating a large package set. One of these "oddities" is that is fairly
```

Reply:
Thanks